crowdsourcing/serializers/project.py

```diff
@@ -36,12 +36,17 @@ class ModuleSerializer(DynamicFieldsModelSerializer):
     deleted = serializers.BooleanField(read_only=True)
     template = TemplateSerializer(many=True, read_only=False)
     module_tasks = TaskSerializer(many=True, read_only=True)
+    csv_data = serializers.ListField(write_only=True)
+    has_data_set = serializers.BooleanField(write_only=True)
```

Comment:
sorry if the comment was confusing I mean it's read and write, that means you dont pass any of the two parameters